index.bs

```diff
@@ -9981,10 +9981,9 @@ Constructors</h5>
 	::
 		When the constructor for {{AudioWorkletProcessor}} is invoked, the following steps are performed:
 		<div algorithm="AudioWorkletProcessor()">
-			1. Compare the value of NewTarget with the <a href=
-				"https://tc39.github.io/ecma262/#active-function-object">active
-				function object</a>; if the two are equal, throw a
-				{{TypeError}}.
+			1. Compare the value of <a href="https://www.ecma-international.org/ecma-262/6.0/#sec-built-in-function-objects">NewTarget</a>
+				with the <a href="https://tc39.github.io/ecma262/#active-function-object">active function object</a>;
```

Comment:
Should both links use the same host? Otherwise, lgtm.

Reply:
The link is not valid any more, so I updated it with the most relevant one from the stable spec. (hosted at ecma-international.org)